examples/annotated-skaffold.yaml

```diff
@@ -183,6 +183,7 @@ deploy:
 
  # helm:
     # helm releases to deploy.
+    # Uses "helm install" if not installed yet, otherwise "helm upgrade" is executed
```

Comment:
So, unfortunately because of the way we have to release config changes, this change also needs to be made in [integration/examples/annotated-skaffold.yaml](https://github.com/GoogleContainerTools/skaffold/blob/master/integration/examples/annotated-skaffold.yaml). Would you mind adding it there as well? Thanks!